pages/project-version/update/submit/index.js

```diff
@@ -98,6 +108,9 @@ module.exports = settings => {
           set(res.locals, `static.content.errors.awerb-${e.id}`, content.errors['awerb-dates']);
         });
         set(res.locals, 'static.content.buttons.submit', get(res.locals, `static.content.buttons.submit.${req.version.type}`));
+        const licenceHolder = req.version.licenceHolder;
+        res.locals.static.licenceHolder = `${licenceHolder.firstName} ${licenceHolder.lastName}`;
+        res.locals.static.type = req.version.type;
```

Comment:
Is this intentional? It doesn't look related to the rest of the PR.

Reply:
I see now, the declaration content has also updated. A little leaky, but whatevs...